### PR TITLE
nutaku: removed largeImageText & formatting

### PR DESCRIPTION
**Pull request by Venipa**

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
Heya @Venipa!
You should check out thing(s) that I requested to change in my review. Please look at them!
 - Bump version.

Thanks,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721

**Review by ririxidev — DISMISSED** [review]
lgtm

**Review by Bas950 — APPROVED** [review]
LGTM

**Comment by ririxidev** [comment]
@Venipa resolve conflicts.

File: websites/N/Nutaku/dist/metadata.json [hunk 1]
```diff
@@ -9,10 +9,14 @@
...
   "regExp": "([a-z]+[.])nutaku.net[/]",
+<<<<<<< HEAD
+  "url": ["nutaku.net", "www.nutaku.net"],
+=======
   "url": [
     "nutaku.net",
     "www.nutaku.net"
   ],
+>>>>>>> 02f5dec3ee4d8fc0e909c353c139d6479182da53
   "version": "1.0.1",
```
Comment (Bas950) [inline websites/N/Nutaku/dist/metadata.json]: doesnt seem right :/

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/N/Nutaku/dist/metadata.json (hunk 1 above)
Comment (Venipa) [reply websites/N/Nutaku/dist/metadata.json]: should be fine now, forgot to add file to git

**Comment by ririxidev** [comment]
bump version again xD

**Comment by Venipa** [comment]
> bump version again xD

1.0.1 is the bumped version

**Comment by Bas950** [comment]
> > bump version again xD
> 
> 1.0.1 is the bumped version
> 
> commit: [f44cc1c](https://github.com/PreMiD/Presences/commit/f44cc1c3d039e85cceeaa32c7221fb679cce7e92)
> smh "files changed" didnt notice it after solving conflicts?

No 1.0.1 is the current version, bump it.

**Review by ririxidev — APPROVED** [review]
lgtm

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Pull request merged**